index.html

```diff
@@ -1238,6 +1238,369 @@ <h2>Evidence</h2>
+			},
+			"proof": {
+				"type": "RsaSignature2018",
+				"created": "2017-06-17T10:03:48Z",
+				"creator": "did:example:ebfeb1f712ebc6f1c276e12ec21/keys/234",
+				"nonce": "d61c4599-0cc2-4479-9efc-c63add3a43b2",
+				"signatureValue": "pYw8XNi1..Cky6Ed = "
+			}
+		}
+	],
+	"proof": [{
+		"type": "RsaSignature2018",
+		"created": "2017-06-18T21:19:10Z",
+		"creator": "did:example:76e12ec21ebhyu1f712ebc6f1z2/keys/2",
+		"nonce": "c0ae1c8e-c7e7-469f-b252-86e6a0e7387e",
+		"signatureValue": "BavEll0/I1..W3JT24 = "
+	}]
+}
+        </pre>
+
+
+   </section>
+  </section>
+
+
+
+   
+   <section>
+      <h2>Holder Acts on Behalf of Subject</h2>
+
+      <p>
+When the holder wants to act on behalf of the subject, the issuer MAY indicate this by
+either including the relationship of the holder in the subject's <a>claim</a>, 
+or by issuing a separate relationship credential to the holder. 
+Either way, the relationship describes the relationship between the holder and the subject of the claim. </p>
+
+<p>It is outside
+the scope of this standard how the issuer verifies the subject, the holder and the
+relationship between them.
+
+<p>
+Note: it is for further study which, if any, of the above mechanisms is to be 
+standardised by the working group.
+
+     </p>
+
+     
+        <pre class="example nohighlight" title="An example of the relationship 
+property in a child's credential, so that a parent can hold it">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "ProofOfAgeCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01T19:73:24Z",
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"ageUnder": 16,
+		"parent": {
+			"type": "Mother",
+			"id": "did:example:ebfeb1c276e12ec211f712ebc6f"
+		}
+	},
+	"proof": {
+		"type": "RsaSignature2018",
+		"created": "2017-06-17T10:03:48Z",
+		"creator": "https://dmv.example.gov/issuers/14/keys/234",
+		"nonce": "d61c4599-0cc2-4479-9efc-c63add3a43b2",
+		"signatureValue": "pY9...Cky6Ed = "
+	}
+} 
+     </pre>
+
+<pre class="example nohighlight" title="An example of a relationship credential issued to
+a parent so that the parent can hold their child's verifiable credentials">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "RelationshipCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01T19:73:24Z",
+	"claim": {
+		"id": "did:example:ebfeb1c276e12ec211f712ebc6f",
+		"child": {
+			"type": ["Person", "Child"],
+			"id": "did:example:ebfeb1f712ebc6f1c276e12ec21"
+		}
+	},
+	"proof": {
+		"type": "RsaSignature2018",
+		"created": "2017-06-17T10:03:48Z",
+		"creator": "https://dmv.example.gov/issuers/14/keys/234",
+		"nonce": "d61c4599-0cc2-4479-9efc-c63add3a43b2",
+		"signatureValue": "pY9...Cky6Ed = "
+	}
+}
+
+</pre>
+     
+<p> It can easily be seen from the above example how an authoritative issuer could indicate that a person is the owner of a pet dog </p>
```

Comment:
While it may be self-explanatory, I feel like this may be the central use case: guardianship (of people, pets, and things).

The delegation bit seems like the wrong way to do things...